Software/src/SettingsWindow.ui

```diff
@@ -720,14 +720,60 @@ Internally emulates the effects of f.lux, redshift, Night Light, Night Shift...<
              <number>0</number>
             </property>
             <item>
-             <widget class="QComboBox" name="comboBox_MoodLampLamp"/>
+             <layout class="QHBoxLayout" name="horizontalLayout_moodlampList">
+              <item>
+               <widget class="QComboBox" name="comboBox_MoodLampLamp"/>
+              </item>
+              <item>
+               <widget class="QToolButton" name="toolButton_MoodLampReload">
+                <property name="text">
+                 <string>Reload</string>
+                </property>
+                <property name="icon">
+                 <iconset resource="../res/LightpackResources.qrc">
+                  <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
+                </property>
+               </widget>
+              </item>
+              <item>
+               <widget class="QToolButton" name="toolButton_MoodLampOpenDir">
+                <property name="text">
+                 <string>Add more</string>
+                </property>
+                <property name="icon">
+                 <iconset resource="../res/LightpackResources.qrc">
+                  <normaloff>:/icons/profile_new.png</normaloff>:/icons/profile_new.png</iconset>
+                </property>
+               </widget>
+              </item>
+              <item>
+               <widget class="QPushButton" name="pushButton_MoodLampOpenReadme">
+                <property name="sizePolicy">
+                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+                  <horstretch>0</horstretch>
+                  <verstretch>0</verstretch>
+                 </sizepolicy>
+                </property>
+                <property name="text">
+                 <string/>
+                </property>
+                <property name="icon">
+                 <iconset resource="../res/LightpackResources.qrc">
+                  <normaloff>:/icons/help.png</normaloff>:/icons/help.png</iconset>
+                </property>
+                <property name="flat">
+                 <bool>true</bool>
+                </property>
+               </widget>
+              </item>
+             </layout>
```

Comment:
Nit: Please add help (hover) texts for the refresh and add buttons (cf. the buttons in the profile tab).

Reply:
Also nit: the tab order should be fixed. I can't currently reach the new elements using tab.